dmriprep/interfaces/vectors.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""@authors: Derek Pisner, Oscar Estaban, Ariel Rokem"""
+from __future__ import division, print_function
```

Comment:
no need for python 2 compatibility (left a comment somewhere else)

Reply:
yep, will remove!